src/scripts/clipperUI/clipper.tsx

```diff
@@ -662,8 +666,10 @@ class ClipperClass extends ComponentBase<ClipperState, {}> {
 		(new OneNoteSaveableFactory(this.state)).getSaveable().then((saveable) => {
 			let saveOptions: SaveToOneNoteOptions = {
 				page: saveable,
-				saveLocation: this.state.saveLocation
+				saveLocation: this.state.saveLocation,
+				progressCallback: this.updateProgress.bind(this)
```

Comment:
Since we don't pass `clipperState` into `SaveToOneNote::save`, I opted to keep this as a callback, but it is easy enough to change.